src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -267,6 +268,8 @@ private async Task<bool> IsIntegratedConsoleEnabledAsync()
                 }
             }
 
+            var commandLineArgs = resolvedProfile.CommandLineArgs != null ? Regex.Replace(resolvedProfile.CommandLineArgs, "[\r\n]+", " ") : null;
```

Comment:
Generally we prefer explicit types over `var`. The exception is when the type is explicit in the initializer.

Reply:
Gotcha, noted.